MurrellThomas_Exercise10.R

```diff
@@ -0,0 +1,64 @@
+# Exercise 10
+# Written by: Isaiah Murrell-Thomas
+
+# Problem 1
+# Using the score-by-score information from this game summarized in “UWvMSU_1-22-13.txt” 
+# generate a graph similar to the one I show above. Don’t worry about how pretty your graph is. 
+# Focus more on the control structures required in your script used to generate the plot.
+# load ggplot2
+library(ggplot2)
+# Data frame set up
+score_data = read.table("UWvMSU_1-22-13.txt", header = TRUE)
+MSU_score = integer(nrow(score_data))
+UW_score = integer(nrow(score_data))
+time = 1:nrow(score_data)
+# Time vector for data frame
+for(i in 1:nrow(score_data)){time[i] = score_data$time[i]}
+# UW and MSU score for data frame
+MSU_score[1] = 0
+UW_score[1] = 3
+for(i in 2:nrow(score_data)){

```

Comment:
You can start with both score equal 0 at time 0 or starting with:
MSU_score[1] = 0
UW_score[1] = 0
the time vector now should include the 0 in the beginning as well